workspaces/meewee-time-registration/plugins/plugin-meewee-time-registration/meewee-time-registration/src/request/api/azdevops/index.ts

```diff
@@ -0,0 +1,2 @@
+export { azDevopsApiRef } from '../../../types/azdevops/types';
+export type { AzDevopsApi } from '../../../types/azdevops/types';
```

Comment:
There are pretty deep paths and I see that throughout the code here. You'll want to follow what is laid out in [ADR004: Module Export Structure](https://backstage.io/docs/architecture-decisions/adrs-adr004)

Reply:
Removed👍